### DISCUSSION TITLE
35F6FF3F-462B-DF4B-160F-812C8B7D2BB5

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#94] [General]
# ESO 510-G13

ESO 510-G13 is an edge-on Sa-type spiral galaxy about 150 million light-years away in the constellation Hydra. Its equatorial dust cloud is heavily warped, indicating it had interacted with another galaxy in the past.

https://celestia.mobi/addon?item=35F6FF3F-462B-DF4B-160F-812C8B7D2BB5&lang=en

**Comment by crystalmirror** [comment]
it's 2d :/ and from the side too

**Reply by TheLostProbe** [reply]
a lot of these billboard models are from decades ago, before Celestia even had proper 3D galaxy rendering. and of course back then the standard for graphics was much lower

**Comment by senormoonfv-sys** [comment]
i love iths bro :)
